File: src/Library/IrCommandStringAttribute.vb
<AttributeUsage(AttributeTargets.Field, AllowMultiple:=False)> _
Public Class IrCommandStringAttribute
    Inherits System.Attribute

    Private _commandString As String

    Public Sub New(ByVal commandString As String)
        _commandString = commandString
    End Sub

    Public ReadOnly Property CommandString() As String
        Get
            Return _commandString
        End Get
    End Property
End Class

File: src/Pixa.Soundbridge/IRCommandStringAttribute.cs
using System;

namespace Pixa.Soundbridge {
    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
    public class IRCommandStringAttribute : Attribute {
        private string _commandString;

        public IRCommandStringAttribute(string commandString) {
            _commandString = commandString;
        }

        public string CommandString {
            get {
                return _commandString;
            }
        }
    }
}